src/compiler/types.ts

```diff
@@ -3055,7 +3055,9 @@ namespace ts {
         ES5 = 1,
         ES6 = 2,

```

Comment:
Or maybe even delete ES6 if it's not used any more. I can't remember if this has anything to do with command line parsing, so it might be hard to tell.